src/current/_includes/molt/fetch-secure-cloud-storage.md

```diff
@@ -0,0 +1,43 @@
+- When using exporting data to [cloud storage]({% link molt/molt-fetch.md %}#cloud-storage), ensure that access control is properly configured:
+
+	- If you are using [Amazon S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/security-iam.html) for cloud storage:
+
+		- Ensure that the environment variable and access tokens are set appropriately in the terminal running `molt fetch`. For example:
+
+			{% include_cached copy-clipboard.html %}
+			~~~ shell
+			export AWS_REGION='us-east-1'
+			export AWS_SECRET_ACCESS_KEY='key'
+			export AWS_ACCESS_KEY_ID='id'
+			export AWS_SESSION_TOKEN='token'
+			export AWS_CREDENTIAL_EXPIRATION='timestamp'
+			~~~
+
+		- Alternatively, set the `--use-implicit-auth` flag to use [implicit authentication]({% link {{ site.current_cloud_version }}/cloud-storage-authentication.md %}).
+
+		- Ensure the S3 bucket is created and accessible by CockroachDB.
+
+	- If you are using [Google Cloud Storage](https://cloud.google.com/storage/docs/access-control) for cloud storage:
+
+		- Ensure that your local environment is authenticated using [Application Default Credentials](https://cloud.google.com/docs/authentication/application-default-credentials):
+
+			Using `gcloud`:
+
+			{% include_cached copy-clipboard.html %}
+			~~~ shell
+			gcloud init
+			gcloud auth application-default login
+			~~~
+
+			Using the environment variable:
+
+			{% include_cached copy-clipboard.html %}
+			~~~ shell
+			export GOOGLE_APPLICATION_CREDENTIALS={path_to_cred_json}
+			~~~
+
+		- Alternatively, set the `--use-implicit-auth` flag to use [implicit authentication]({% link {{ site.current_cloud_version }}/cloud-storage-authentication.md %}).
+
+		- Ensure the Google Cloud Storage bucket is created and accessible by CockroachDB.
+
+	- Do not use public cloud storage in production.
```

Comment:
We want customers to use public cloud storage in production though. Its more resilient. Unless you were trying to convey something else here.

Reply:
This is an artifact from earlier docs, but I think it might have meant "publicly accessible". Would that be a better phrase?

Reply:
Ohh I see what you're getting at. Meaning like external people (bad people on the internet) can't find and access the bucket.

Reply:
I think this could be combined with the bullet above maybe and state that "Ensure the GCS bucket is created and accessible by authorized roles/users only"

Maybe something to that effect

Reply:
Updated.